api/src/shop/views.py

```diff
@@ -334,3 +336,23 @@ def download_accounting_file_route(year: str, month: str):
     return b64encode(export_accounting(start_date, end_data, TimePeriod.Month, g.user_id).encode("cp437")).decode(
         "ascii"
     )
+
+
+@service.route("/product/<int:product_id>/sales", method=GET, permission=WEBSHOP)
+def product_sales_route(product_id: int) -> Any:
+    start_str = request.args.get("start", default=None)
+    start = datetime.fromisoformat(start_str) if start_str is not None else None
+
+    end_str = request.args.get("end", default=None)
+    end = datetime.fromisoformat(end_str) if end_str is not None else None
+    return product_sales(product_id, start, end).to_dict()
+
+
+@service.route("/category/<int:category_id>/sales", method=GET, permission=WEBSHOP)
+def category_sales_route(category_id: int) -> Any:
+    start_str = request.args.get("start", default=None)
+    start = datetime.fromisoformat(start_str) if start_str is not None else None
```

Comment:
`fromisoformat` might differ in what time it considers the end of day compared to the bookkeeping? I think the bookkeeping always considers the "local" timezone, while IIUC this is UTC.

If so, I don't think it's anything major, but just worth noting in case the accountant starts using this tool and wondering why there might be a small diff.

Reply:
fromisoformat will use the local timezone unless specified.
For example: `start=2024-01-01` will return "2024-01-01 00:00" in the local timezone, while
`start=2024-01-01 00:00:00+00:00` will return "2024-01-01 00:00" in the UTC timezone.

Reply:
Also. The server's local timezone is UTC+0. So I honestly don't think we are helped that much by interpreting things in the local timezone.